spec/runtime/with_clean_env_spec.rb

```diff
@@ -3,8 +3,9 @@
 RSpec.describe "Bundler.with_env helpers" do
   describe "Bundler.original_env" do
     before do
+      bundle "config path vendor/bundle"
       gemfile ""
-      bundle "install --path vendor/bundle"
+      bundle "install"
     end
 
     it "should return the PATH present before bundle was activated" do
```

Comment:
I think I fixed this in another PR but can't remember. https://github.com/bundler/bundler/pull/5720/commits/d3e08aaa44645450af025f176e3c31d58c51e8ec